packages/conf-libssl/conf-libssl.3/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "David Sheets <sheets@alum.mit.edu>"
+authors: ["The OpenSSL Project"]
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://www.openssl.org/"
+license: "Apache-1.0"
+build: [
+  ["pkg-config" "--print-errors" "--exists" "openssl"]
+    {os != "freebsd" & os != "openbsd" & os != "netbsd" & # libssl is provided by base system on BSDs
+     os-distribution != "homebrew" & os-distribution != "macports" & os-distribution != "nixos"}
+  ["sh" "-c" "env \"PKG_CONFIG_PATH=$(brew --prefix openssl)/lib/pkgconfig:$PKG_CONFIG_PATH\" pkg-config --print-errors --exists openssl"] {os-distribution = "homebrew"}
+  ["sh" "-c" "env \"PKG_CONFIG_PATH=$HOME/.nix-profile/lib/pkgconfig:$PKG_CONFIG_PATH\" pkg-config --print-errors --exists openssl"] {os-distribution = "nixos"}
+  ["sh" "-c" "env \"PKG_CONFIG_PATH=/opt/local/lib/pkgconfig:$PKG_CONFIG_PATH\" pkg-config --print-errors --exists openssl"] {os-distribution = "macports"}
```

Comment:
I never needed to add this to compile against libraries installed by macports, I think PKG_CONFIG_PATH is set up correctly by macports when you install the distribution

Reply:
I am creating a fresh environment to check this. It will take a while :P

Reply:
That's also my experience. MacPorts doesn't care to hide system tools if MacPorts binaries are in the PATH. Homebrew installs to a path which is in PATH by default, so it needs to take care to not hide system tools. The MacPorts pkg-config takes the packages installed by MacPorts into account.